packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/cloudwatch_logger_plugin.dart

```diff
@@ -126,65 +126,70 @@ class CloudWatchLoggerPlugin extends AWSLoggerPlugin
   }
 
   Future<void> _startSyncingIfNotInProgress() async {
-    Future<void> startSyncing() async {
-      final batchStream = _getLogBatchesToSync();
-      await for (final (logs, events) in batchStream) {
-        _TooNewLogEventException? tooNewException;
-        while (logs.isNotEmpty && events.isNotEmpty) {
-          final rejectedLogEventsInfo =
-              (await _sendToCloudWatch(events)).rejectedLogEventsInfo;
-          if (rejectedLogEventsInfo == null) {
-            await _logStore.deleteItems(logs);
-            break;
+    await identifyCall(LoggingCategoryMethod.batchSend, () async {
```

Comment:
issue: this is called by flush api, timer sync and logStore full sync. instead of wrapping this we should use identifyCall where we call `_startSyncingIfNotInProgress` like:

```
_timer = pluginConfig.flushIntervalInSeconds > 0
        ? StoppableTimer(
            duration: Duration(seconds: pluginConfig.flushIntervalInSeconds),
            callback: identifyCall( LoggingCategoryMethod.batchSend,
                () => _startSyncingIfNotInProgress,),
            onError: _onTimerError,
          )
```

```
if (isLogStoreFull && _shouldSyncOnFullLogStore()) {
      await identifyCall( LoggingCategoryMethod.batchSend,
          _startSyncingIfNotInProgress);
    }
```